composer.json

```diff
@@ -1,39 +1,19 @@
 {
     "name": "typo3/reference-coreapi",
-    "type": "typo3-cms-documentation",
     "description": "Main TYPO3 core documentation",
     "license": "OPL-1.0",
+    "type": "typo3-cms-documentation",
     "require": {
-      "typo3/cms-core": "dev-main as 12.0"
-    },
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "extra": {
-        "typo3/cms": {
-            "web-dir": ".Build/public",
-            "extension-key": "t3docs-reference-coreapi"
-        }
-    },
-    "repositories": {
-        "1a_git": {
-            "type": "vcs",
-            "url": "https://github.com/TYPO3-Documentation/tea.git"
-        },
-        "1c_git": {
-            "type": "vcs",
-            "url": "https://github.com/TYPO3-Documentation/blog_example.git"
-        }
-    },
-    "config": {
-        "allow-plugins": {
-            "typo3/cms-composer-installers": true,
-            "typo3/class-alias-loader": true
-        },
-        "vendor-dir": ".Build/vendor"
+        "typo3/cms-core": "dev-main as 12.0"
     },
     "require-dev": {
-        "t3docs/codesnippet": "dev-main",
+        "rector/rector": "0.14.6",
         "t3docs/blog-example": "dev-main",
```

Comment:
Can be "^12.0".

Reply:
can we change that in a follow up? Out of scope of this PR

Reply:
Yup, go ahead.